fonts/generate-smufl-code.py

```diff
@@ -2,10 +2,11 @@
 
 import os
 import xml.etree.ElementTree as ET
+from typing import Dict
```

Comment:
PEP585 deprecated these type hints as of Python 3.9; is there a minimum Python version requirement?